stock_container_transit/models/freight_container.py

```diff
@@ -0,0 +1,18 @@
+from odoo import fields, models
+
+
+class FreightContainer(models.Model):
+    _name = "freight.container"
+    _description = "Goods container"
+
+    freight_id = fields.Many2one(comodel_name="freight")
```

Comment:
should be an ordered many2many ?

because you may want to manage a multistep route and or reuse the same container for multiple trips

Reply:
I now think that freight.container and freight objects must be linked in the picking independently as many2one.
I 'll make the change. What do you think @PaulGoubert @hparfr